maps/biter_battles_v2/tables.lua

```diff
@@ -32,17 +32,18 @@ Public.upgrade_modifiers = {
 }
 
 Public.food_values = {
-    ['automation-science-pack'] = { value = 0.0009, name = 'automation science', color = '255, 50, 50' },
-    ['logistic-science-pack'] = { value = 0.0023, name = 'logistic science', color = '50, 255, 50' },
-    ['military-science-pack'] = { value = 0.0095, name = 'military science', color = '105, 105, 105' },
-    ['chemical-science-pack'] = { value = 0.0250, name = 'chemical science', color = '100, 200, 255' },
-    ['production-science-pack'] = { value = 0.1050, name = 'production science', color = '150, 25, 255' },
-    ['utility-science-pack'] = { value = 0.1680, name = 'utility science', color = '210, 210, 60' },
-    ['space-science-pack'] = { value = 0.3500, name = 'space science', color = '255, 255, 255' },
-    ['metallurgic-science-pack'] = { value = 0.0010, name = 'metallurgic science', color = '255, 165, 0' },
-    ['electromagnetic-science-pack'] = { value = 0.0011, name = 'electromagnetic science', color = '255, 192, 203' },
-    ['agricultural-science-pack'] = { value = 0.0012, name = 'agricultural science', color = '192, 255, 0' },
-    ['cryogenic-science-pack'] = { value = 0.0013, name = 'cryogenic science', color = '0, 0, 139' },
+    ['automation-science-pack'] = { value = 0.001, name = 'automation science', color = '255, 50, 50' },
+    ['logistic-science-pack'] = { value = 0.0025, name = 'logistic science', color = '50, 255, 50' },
+    ['military-science-pack'] = { value = 0.009, name = 'military science', color = '105, 105, 105' },
+    ['chemical-science-pack'] = { value = 0.012, name = 'chemical science', color = '100, 200, 255' },
+    ['production-science-pack'] = { value = 0.04, name = 'production science', color = '150, 25, 255' },
+    ['utility-science-pack'] = { value = 0.07, name = 'utility science', color = '210, 210, 60' },
+    ['space-science-pack'] = { value = 0.003, name = 'space science', color = '255, 255, 255' },
```

Comment:
Order is different here. Shouldn't matter as it's a keyed table, but could be worth for consistency ensure order is same everywhere so nothing is missed.

References were scattered across codebase, so just trying to be thorough, as science packs have never changed previously.